README.md

```diff
@@ -85,6 +85,10 @@ Contributions are welcome!
 3. We recommend to write a test which shows that the bug was fixed or that the feature works as expected.
 4. Send a pull request and bug the maintainer until it gets merged and published. :smiley:
 
+## Fix issue there're some applications that launched process exists immediately(Ex: microsoft calculator, firefox, chrome, iexplorer), so in close/quit function it throws exception Process Not Found.
```

Comment:
from @zelonght 
(All these changes) This should be somewhere like changelog, not in README.md